.build/docker/run-tests.sh

```diff
@@ -0,0 +1,259 @@
+#!/bin/bash
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#
+#
+# A wrapper script to run-tests.sh (or dtest-python.sh)
+#  that splits the test list into multiple docker runs, collecting results.
+#
+# Each split chunk may be further parallelised over docker containers based on the host's available cpu and memory.
```

Comment:
Is this required because we have a heterogenous execution environment? And if so, will that remain true going forward?

Even if not, the ability to disable / bypass this step might be useful in the case where something is executing in a heterogenous environment.

Reply:
yes. and quite likely yes (currently is disabled for this reason).